### DISCUSSION TITLE
MessageBody中SoraSegment.xxxx能否默认换行？

### DISCUSSION THREAD

**Discussion by ShellyMon** [Hoshikawa-Kaguya/Sora#88] [Q&A]
![image](https://github.com/Hoshikawa-Kaguya/Sora/assets/48594597/6bdeeedd-b15a-4ec9-bac5-8459693f689f)

**Comment by ShellyMon** [comment]
我认为这样会美观一点

**Reply by Yukari316** [reply]
不默认换行，因为容易造成错误理解
